snapcraft/commands/daemon.start

```diff
@@ -442,20 +442,6 @@ if [ "$(stat -c '%u' /proc)" = 0 ]; then
             echo 1 > /proc/sys/kernel/unprivileged_userns_clone || true
         fi
     fi
-
-    if [ -e /proc/sys/kernel/apparmor_restrict_unprivileged_userns ]; then
```

Comment:
do we need this commit, they should remain globally disabled in my view for Noble